keyboards/keebfront/vanguard65/keyboard.json

```diff
@@ -0,0 +1,743 @@
+{
+    "manufacturer": "Keebfront",
+    "keyboard_name": "Vanguard65",
+    "maintainer": "Keebfront",
+    "bootloader": "atmel-dfu",
+    "build": {
+        "lto": true
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "E6", "pin_b": "D4"}
+        ]
+    },
+    "features": {
```

Comment:
```suggestion
    "features": {
        "bootmagic": true,
```

Reply:
Interesting, did Bootmagic get stripped from the default behavior? I always thought it was included as default.

Reply:
Via does enable it by default, but otherwise, no it's not enabled.

Reply:
Good to know! Thank you!